src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1449,6 +1449,24 @@
   <data name="UseEnvironmentCurrentManagedThreadIdFix" xml:space="preserve">
     <value>Use 'Environment.CurrentManagedThreadId'</value>
   </data>
+  <data name="ThreadStaticOnNonStaticFieldDescription" xml:space="preserve">
+    <value>'ThreadStatic' only affects static fields.  When applied to instance fields, it has no impact on behavior.</value>
```

Comment:
This looks like an extra space

```suggestion
    <value>'ThreadStatic' only affects static fields. When applied to instance fields, it has no impact on behavior.</value>
```

Reply:
I removed the space, as I assume from your comment two-spaces-after-a-period is not the style used in analyzers.  It's just what I've always done :)